### ISSUE TITLE
Pitch bend doesn't work properly in IT2MID

### ISSUE THREAD

**Issue by turboboy215** [ValleyBell/MidiConverters#6]
I have recently been using your IT2MID program, and I must say it's been working well. The MIDI files it outputs are "more optimized" than those generated from OpenMPT's export function. However, one thing is incorrect that even OpenMPT sort of gets right: the pitch bend function when converting Exx and Fxx commands, which some of my songs that I made have a lot of. It seems that it's mapped to the "modulation" command rather than pitch bend when I open it in a MIDI editor, and doesn't seem to change pitch very much at all. All of your other MIDI converters don't have this issue. What is happening here? Is it a bug or is there a way to get these pitch slides right?

**Comment by ValleyBell** [comment]
I'm afraid that the Pitch Slide commands are indeed mapped to General Purpose 3/4 + Modulation.

it2mid just converts all events very literally. It doesn't have an actual per-tick channel processor, which it would need in order to convert pitch slide commands into actual pitch bends.

tl;dr it's not a bug, but a limitation of the tool.

**Comment by turboboy215** [comment]
How would I accurately convert my Its to MIDI then with proper pitch bend? Would you have to re-write the tool? Do I have to use OpenMPT’s feature instead?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by ValleyBell** [comment]
Yes, you will have to use OpenMPT's exporter or maybe LifeAmp. Actual pitch bend support would require me to rewrite major parts of the tool.

**Comment by ValleyBell** [comment]
I'm closing this with *won't fix* as right now I don't intend to rewrite it2mid.

Pull requests are welcome though.
